homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html

```diff
@@ -32,12 +32,27 @@
                     <div class="return">
                         <button type="button" class="button">Calculation</button>
                     </div>
-                    <div class="re">
-                        <p class="resul__number result"></p>
+                    <div class="count-block">
+                        <div class="visual-counting">
+                            <div class="number__one">
+                                <h2 class="number__1"></h2>
+                            </div>
+                            <div class="operator__visual">
+                                <h2 class="oper"></h2>
+                            </div>
+                            <div class="number__two">
+                                <h2 class="number__2"></h2>
+                            </div>
+                            <div class="equals"></div>
+                        </div>
+                        <div class="result__block">
+                            <p class="resul__number result"></p>
+                        </div>
                     </div>
                 </div>
             </main>
         </div>
         <script src="homework.js"></script>
+        <script src="calculator.js"></script>
```

Comment:
Are you sure you need both of these scripts?
On each file you declared a variable with the equal name 'operator' and ' btn', so your logic doesn't work as you have an error in the console, please fix it:
<img width="1414" alt="Screenshot 2023-11-07 at 01 59 12" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/118203044/a78eb8c7-7533-45c7-9090-4aefd71fe113">